pkg/controller/cloud/nodecontroller.go

```diff
@@ -55,6 +65,13 @@ const (
 
 	// The amount of time the nodecontroller should sleep between retrying NodeStatus updates
 	retrySleepTime = 20 * time.Millisecond
+
+	//Taint denoting that a node needs to be processed by external cloudprovider
+	CloudTaintKey = "ExternalCloudProvider"
+
+	nodeStatusUpdateFrequency = 10 * time.Second
+
+	LabelProvidedIPAddr = "beta.kubernetes.io/provided-node-ip"
```

Comment:
Kubelet held it memory before. Now, I'm using a label instead of an annotation because, this information can be used to uniquely identify the resource. 

I wonder what the use case for this field is. It was here originally - https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/options/options.go#L136